src/sherpa_ai/citation_validation.py

```diff
@@ -0,0 +1,115 @@
+import nltk
```

Comment:
Let's write this in the form of an [output_parser](https://github.com/Aggregate-Intellect/sherpa/blob/main/src/sherpa_ai/output_parsers/base.py) so that we can use it later on.

Reply:
Do it now